client/src/components/pages/RobotOverview/RobotOverview.jsx

```diff
@@ -60,6 +61,23 @@ const RobotOverview = () => {
     setSearchValue(value);
   };
 
+  /**
+ * @description Creates a new bot for the current userId
+ */
+  const createNewRobot = () => {
+    const robotName = 'New Robot';
+    createNewBot(userId, robotName)
+      .then((response) => response.json())
+      .then((data) => {
+        const newList = [...robotList]
+        newList.push(data);
```

Comment:
can't we push the new bot to the beginning of the list? I would prefer the new bot to pop up right next to the "add new bot" box. Imagine you have 100 bots and to create a new bot you would have to always scroll all the way down

Reply:
can you please also rename `data`

Reply:
+1

Reply:
@lukashueller https://www.w3schools.com/jsref/jsref_unshift.asp

Reply:
@SanJSp I was unfortunately not able to fix this even after half an hour of research. `Unshift()` already does what it is supposed to; however, React only re-renders the last box, which is why we don't get the desired result.